assets/css/style.css

```diff
@@ -0,0 +1,61 @@
+/* Just some random styles to help show the bare html layout generated so far */
+body {
+    text-align: center;
+}
+
+.container {
+    min-height: 100vh;
+    margin: -60px;
```

Comment:
These need to be changed to margin-bottom, padding-bottom.

Reply:
Done!